go-controller/pkg/ovn/ovn.go

```diff
@@ -282,3 +449,18 @@ func (oc *Controller) WatchNodes() error {
 	}, nil)
 	return err
 }
+
+// AddServiceVIPToName associates a k8s service name with a load balancer VIP
+func (oc *Controller) AddServiceVIPToName(vip string, protocol kapi.Protocol, namespace, name string) {
+	oc.serviceVIPToNameLock.Lock()
+	defer oc.serviceVIPToNameLock.Unlock()
+	oc.serviceVIPToName[ServiceVIPKey{vip, protocol}] = types.NamespacedName{namespace, name}
+}
+
+// GetServiceVIPToName retrieves the associated k8s service name for a load balancer VIP
+func (oc *Controller) GetServiceVIPToName(vip string, protocol kapi.Protocol) (types.NamespacedName, bool) {
+	oc.serviceVIPToNameLock.Lock()
+	defer oc.serviceVIPToNameLock.Unlock()
+	namespace, ok := oc.serviceVIPToName[ServiceVIPKey{vip, protocol}]
```

Comment:
just `return oc.serviceVIPToName[ServiceVIPKey{vip, protocol}]`

Reply:
Unfortunately go complains about this:

# github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn
pkg/ovn/ovn.go:471:2: not enough arguments to return
	have ("github.com/ovn-org/ovn-kubernetes/go-controller/vendor/k8s.io/apimachinery/pkg/types".NamespacedName)
	want ("github.com/ovn-org/ovn-kubernetes/go-controller/vendor/k8s.io/apimachinery/pkg/types".NamespacedName, bool)
make: *** [Makefile:20: all] Error 2